tests/core/math/test_vector3.h

```diff
@@ -122,6 +130,9 @@ TEST_CASE("[Vector3] Interpolation methods") {
 	CHECK_MESSAGE(
 			Vector3(1, 1, 1).slerp(Vector3(), 0.5) == Vector3(0.5, 0.5, 0.5),
 			"Vector3 slerp with one input as zero should behave like a regular lerp.");
+	CHECK_MESSAGE(
+			Vector3(4, 6, 2).slerp(Vector3(8, 10, 3), 0.5).is_equal_approx(Vector3(5.90194, 8.06759, 2.55831)),
```

Comment:
Like a madman I computed the entire slerp process manually in that high precision calculator. I won't include this many digits when I push but I just wanted to share the full result in a comment (it's probably accurate to 90+ digits).

```
5.9019421981142994105328592245732199235862625080945286020815665253148024689834897464074096522087271
8.0675868884937839453430498266678652230734384658846145797809906007724742838385711767541311745863104
2.5583078947183171200383101071906276306401536059186751393701036690575365246734131517752131742409734
```

Reply:
This had got to be the most tedious yet immensely timesaving task that future Godot devs are going to thank you for.